src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServices.cs

```diff
@@ -8,14 +8,14 @@ namespace NuGet.VisualStudio.Internal.Contracts
 {
     public static class NuGetServices
     {
-        public static string NuGetSolutionServiceName = "NuGetSolutionService";
-        public static string NuGetSolutionServiceVersion = "1.0.0";
+        private const string NuGetSolutionServiceName = "NuGetSolutionService";
+        private const string NuGetSolutionServiceVersion = "1.0.0";
```

Comment:
suggestion: add a comment saying that if these are changed, then `src/NuGet.Clients/NuGet.SolutionRestoreManager/BrokeredServicesUtility.cs` should be updated too.

Reply:
@dominoFire recommended adding a test, would you prefer that over a comment? 
Thoughts on adding the test in the first place? (might do a different PR, don't want to complicate things here).

Reply:
A comment or a test (or both) is fine in another PR, as long as we have something to remember us to sync these values.